pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -76,43 +76,44 @@ func (m *MultiNodeConsolidation) ComputeCommand(ctx context.Context, disruptionB
 	// This could be further configurable in the future.
 	maxParallel := lo.Clamp(len(disruptableCandidates), 0, 100)
 
-	cmd, err := m.firstNConsolidationOption(ctx, disruptableCandidates, maxParallel)
+	cmd, results, err := m.firstNConsolidationOption(ctx, disruptableCandidates, maxParallel)
 	if err != nil {
-		return Command{}, err
+		return Command{}, nil, err
 	}
 
 	if cmd.Action() == NoOpAction {
 		// couldn't identify any candidates
 		m.markConsolidated()
-		return cmd, nil
+		return cmd, nil, nil
 	}
 
 	v := NewValidation(consolidationTTL, m.clock, m.cluster, m.kubeClient, m.provisioner, m.cloudProvider, m.recorder, m.queue)
 	isValid, err := v.IsValid(ctx, cmd)
 	if err != nil {
-		return Command{}, fmt.Errorf("validating, %w", err)
+		return Command{}, nil, fmt.Errorf("validating, %w", err)
 	}
 
 	if !isValid {
 		logging.FromContext(ctx).Debugf("abandoning multi-node consolidation attempt due to pod churn, command is no longer valid, %s", cmd)
-		return Command{}, nil
+		return Command{}, nil, nil
 	}
-	return cmd, nil
+	return cmd, results, nil
 }
 
 // firstNConsolidationOption looks at the first N NodeClaims to determine if they can all be consolidated at once.  The
 // NodeClaims are sorted by increasing disruption order which correlates to likelihood if being able to consolidate the node
-func (m *MultiNodeConsolidation) firstNConsolidationOption(ctx context.Context, candidates []*Candidate, max int) (Command, error) {
+func (m *MultiNodeConsolidation) firstNConsolidationOption(ctx context.Context, candidates []*Candidate, max int) (Command, *scheduling.Results, error) {
 	// we always operate on at least two NodeClaims at once, for single NodeClaims standard consolidation will find all solutions
 	if len(candidates) < 2 {
-		return Command{}, nil
+		return Command{}, nil, nil
 	}
 	min := 1
 	if len(candidates) <= max {
 		max = len(candidates) - 1
 	}
 
 	lastSavedCommand := Command{}
+	results := &scheduling.Results{}
```

Comment:
```suggestion
	lastSavedResults := &scheduling.Results{}
```

Should we call this `lastSavedResults` for parity and then we can just call it `results` in the binary search?

Reply:
Good callout!